engine/api/pipeline_import.go

```diff
@@ -16,6 +16,80 @@ import (
 	"github.com/ovh/cds/sdk/exportentities"
 )
 
+func (api *API) postPipelinePreviewHandler() Handler {
+	return func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+		format := r.FormValue("format")
+
+		// Get body
+		data, errRead := ioutil.ReadAll(r.Body)
+		if errRead != nil {
+			return sdk.WrapError(sdk.ErrWrongRequest, "postPipelinePreviewHandler> Unable to read body : %v", errRead)
+		}
+
+		// Compute format
+		f, errF := exportentities.GetFormat(format)
+		if errF != nil {
+			return sdk.WrapError(sdk.ErrWrongRequest, "postPipelinePreviewHandler> Unable to get format : %v", errF)
+		}
+
+		rawPayload := map[string]interface{}{}
+		var errorParse error
+		switch f {
+		case exportentities.FormatJSON:
+			errorParse = json.Unmarshal(data, &rawPayload)
+		case exportentities.FormatYAML:
+			errorParse = yaml.Unmarshal(data, &rawPayload)
+		}
+
+		if errorParse != nil {
+			return sdk.NewError(sdk.ErrWrongRequest, errorParse)
+		}
+
+		//Parse the data once to retrieve the version
+		var pipelineV1Format bool
+		if v, ok := rawPayload["version"]; ok {
```

Comment:
i think it's not very useful to support the old pipeline format `exportentities.Pipeline`. Just keep `exportentities.PipelineV1`